include/COLLADA2GLTFWriter.h

```diff
@@ -10,10 +10,15 @@
 #include "COLLADA2GLTFExtrasHandler.h"
 
 #include "draco/compression/encode.h"
+#include "draco/core/draco_types.h"
 
 namespace COLLADA2GLTF {
 	class Writer : public COLLADAFW::IWriter {
 	private:
+		typedef uint32_t index_t;
+		typedef int32_t signed_index_t;
```

Comment:
Why does this exist? Indices are never signed.

In general, I would prefer not to macro out types like this since I think it lowers the readability of the code, especially when there are only three possible values anyway via the spec.

> 5121 (UNSIGNED_BYTE), 5123 (UNSIGNED_SHORT) or 5125 (UNSIGNED_INT)

I think the preferred implementation would be to drop the command line option, and precompute the required number of indices during mesh generation, then choose the appropriate type for the container (probably excluding UNSIGNED_BYTE)